docs/components/CreateNftSection.tsx

```diff
@@ -197,14 +197,20 @@ const ImageDropZone = () => {
   const { values, setFieldValue } = useFormikContext();
   const data = values as FormData;
 
+  const [uploading, setUploading] = useState(false);
+
   const { getRootProps, getInputProps, isDragActive } = useDropzone({
     accept: ACCEPT_IMAGE_PROP,
     multiple: false,
     onDrop: async (files) => {
       const [file] = files;
-      const { file_url } = await uploadImageToS3({ file });
 
+      setUploading(true);
+
+      const { file_url } = await uploadImageToS3({ file });
```

Comment:
This should probably by in a try finally. If upload fails it'll get stuck in loading state